docker/docker-compose.yml

```diff
@@ -39,4 +39,4 @@ networks:
   emission:
 
 volumes:
-  mongo-data:
\ No newline at end of file
+  mongo-data:
```

Comment:
why do you need this?

Reply:
Not sure what you mean. why do I need the mongo-data volume?